projects/v1/default.wmf.yaml

```diff
@@ -96,3 +96,5 @@ paths:
       - path: v1/javascript.yaml
         options:
           host: '{{options.mobileapps.host}}'
+      - path: v1/recommend-caption.yaml
+        options: '{{options.appeditortasks}}'
```

Comment:
Mmmm... Haven't we decided that this will only be exposed on commons?

Reply:
Done.

Reply:
> Mmmm... Haven't we decided that this will only be exposed on commons?

Er, that's news to me. Was that discussed in the sync meeting last week?  The description suggestion endpoints don't make sense on Commons.

Reply:
Never mind, misunderstood the comment ;)